### PR TITLE
Fix entity_attention_mask for NER.

### PR DESCRIPTION
**Pull request by chantera**
Related to #166, this PR fixes `entity_attention_mask` for `examples/ner/evaluate_transformers_checkpoint.py`.

### CODE REVIEW

**Comment by Ryou0634** [comment]
Thank you for fixing this problem!
The PR looks good to me.

We will try fine-tuning models with the correct padding and update the model with a better one!
